fwoa-tools/src/export-script.py

```diff
@@ -61,210 +67,214 @@
-else:
-    print('Loading patient compartment search params')
-    client = boto3.client('s3')
-    s3Obj_compartment = client.get_object(Bucket = s3_script_bucket,
-                Key = compartment_search_param_file)
-    compartment_search_params = json.load(s3Obj_compartment['Body'])
-
-    s3Obj_transitive = client.get_object(Bucket = s3_script_bucket,
-                    Key = transitive_reference_param_file)
-    transitive_reference_params = json.load(s3Obj_transitive['Body'])
-
-    print('Extract group member ids')
-    group_members = Filter.apply(frame = filtered_tenant_id_frame, f = lambda x: x['id'] == group_id).toDF().sort("meta.versionId").collect()[-1]['member']
-    active_group_member_references = [x['entity']['reference'] for x in group_members if is_active_group_member(x, datetime_transaction_time) and is_internal_reference(x['entity']['reference'], server_url)]
-    group_member_ids = set([x.split('/')[-1] for x in active_group_member_references])
-    group_patient_ids = set([x.split('/')[-1] for x in active_group_member_references if x.split('/')[-2] == 'Patient'])
-    print('Group member ids extracted: ', group_member_ids)
-    print('Group patient ids extracted: ', group_patient_ids)
-
-    print('Extract group member and patient compartment dataframe')
-    filtered_group_frame = Filter.apply(frame = filtered_tenant_id_frame, f = lambda x: is_included_in_group_export(x, group_member_ids, group_patient_ids, compartment_search_params, server_url))
-
-    filtered_group_reference_frame = filtered_group_frame
-    if transitive_reference_params:
-        print('Extract transitive references')
-        transitive_reference_frame = Map.apply(frame = filtered_group_frame, f = lambda x: get_transitive_references(x, transitive_reference_params, server_url))
-        transitive_reference_frame = Filter.apply(frame = transitive_reference_frame, f = lambda x: x['_generated_transitive_refs'] is not None)
-        transitive_reference_frame = SelectFields.apply(frame = transitive_reference_frame, paths=['_generated_transitive_refs']).toDF().collect()
-
-        transitive_reference_set = set()
-        for item in transitive_reference_frame:
-            transitive_reference_set.update(reference.split('/')[-1] for reference in item['_generated_transitive_refs'])
-        print('Transitive reference ids extracted: ', transitive_reference_set)
-
-        if transitive_reference_set:
-            # Filter here again from tenant_id_frame to include group members, patient compartment and transitive reference
-            filtered_group_reference_frame = Filter.apply(frame = filtered_tenant_id_frame, f = lambda x: is_included_in_group_export(x, group_member_ids, group_patient_ids, compartment_search_params, server_url, transitive_reference_ids=transitive_reference_set))
+table_export_finish_time = datetime.now()
+print('Elapsed time for tenantId Filtering = ',
+      table_export_finish_time - table_export_start_time)
```

Comment:
We should probably use different variables for the different bench marks that we're doing. IN this case it would be `filter_start_time` and `filter_finish_time`.